ocs_ci/ocs/resources/osd_resize.py

```diff
@@ -0,0 +1,306 @@
+    current_pvc_names = [p.name for p in current_osd_pvcs]
+    logger.info(f"Old PVC names = {old_pvc_names}")
+    logger.info(f"Current PVC names = {current_pvc_names}")
+
+    current_osd_pvs = get_deviceset_pvs()
+    old_pv_names = [p.name for p in old_osd_pvs]
+    current_pv_names = [p.name for p in current_osd_pvs]
+    logger.info(f"Old PV names = {old_pv_names}")
+    logger.info(f"Current PV names = {current_pv_names}")
+
+    logger.info(
+        "Check that the old PVC and PV names are equal to the current PVC and PV names"
+    )
+    if not old_pvc_names == current_pvc_names:
+        raise ResourceWrongStatusException(
+            f"The old PVC names {old_pvc_names} are not equal to the "
+            f"current PVC names {current_pvc_names}"
+        )
+    if not old_pv_names == current_pv_names:
+        raise ResourceWrongStatusException(
+            f"The old PV names {old_pv_names} are not equal to the "
+            f"current PV names {current_pv_names}"
+        )
+
+
+def check_storage_size_is_reflected(expected_storage_size):
+    """
+    Check that the expected storage size is reflected in the current storage size, PVCs, PVs,
+    and ceph capacity.
+
+    Args:
+        expected_storage_size (str): The expected storage size
+
+    Raises:
+        StorageSizeNotReflectedException: If the current storage size, PVCs, PVs, and ceph capacity
+            are not in the expected size
+
+    """
+    logger.info(f"The expected storage size is {expected_storage_size}")
+
+    current_storage_size = get_storage_size()
+    logger.info(f"The current storage size is {current_storage_size}")
+    logger.info(
+        "Check that the current storage size equal to the expected storage size"
+    )
+    if get_storage_size() != expected_storage_size:
+        raise StorageSizeNotReflectedException(
+            f"The current storage size {current_storage_size} is not equal "
+            f"to the expected size {expected_storage_size}"
+        )
+
+    logger.info(
+        "Check that the PVC and PV sizes are equal to the expected storage size"
+    )
+    current_osd_pvcs = get_deviceset_pvcs()
+    expected_storage_size_in_gb = convert_device_size(expected_storage_size, "GB")
+    pvc_sizes = [pvc.size for pvc in current_osd_pvcs]
+    logger.info(f"PVC sizes = {pvc_sizes}")
+    if not all([p_size == expected_storage_size_in_gb for p_size in pvc_sizes]):
+        raise StorageSizeNotReflectedException(
+            f"The PVC sizes are not equal to the expected storage size {expected_storage_size_in_gb}"
+        )
+
+    current_osd_pvs = get_deviceset_pvs()
+    pv_sizes = [get_pv_size(pv.get()) for pv in current_osd_pvs]
+    logger.info(f"PV sizes {pv_sizes}")
+    if not all([p_size == expected_storage_size_in_gb for p_size in pv_sizes]):
+        raise StorageSizeNotReflectedException(
+            f"The PV sizes are not equal to the expected storage size {expected_storage_size_in_gb}"
```

Comment:
```suggestion
            f"The PV sizes are not equal to the expected storage size {expected_storage_size_in_gb}. "
            f"PV sizes: {[p_size for p_size in pv_sizes]}"
```

Reply:
Here, it logs the PV sizes anyway a few lines back. Should we still add this log message?